server/src/main/java/org/cloudfoundry/identity/uaa/scim/validate/UaaPasswordPolicyValidator.java

```diff
@@ -62,7 +62,7 @@ public void validate(String password) throws InvalidPasswordException {
             policy = idpDefinition.getPasswordPolicy();
         }
 
-        org.passay.PasswordValidator validator = validator(policy, messageResolver);
```

Comment:
no logic change here, only renamed it to make clear, we have users here

Reply:
When you feel like you have to clarify things in github comments, IMO it means that you should have clarified them in the code. Or through git commits.

For example, what I would have done in this specific case, is to made a separate commit for the renaming of the function that I would title: "refactor: rename function". This way it is obvious to everyone that it is a refactor. Not only to people reading the PR.